hail/python/hailtop/hailctl/batch/cli.py

```diff
@@ -162,3 +163,11 @@ def submit(
 ):
     '''Submit a batch with a single job that runs SCRIPT with the arguments ARGUMENTS.'''
     asyncio.run(_submit.submit(name, image_name, files or [], output, script, arguments or []))
+
+
+@app.command('init', help='Initialize a Hail Batch environment.')
+def initialize(
+        domain: Ann[Optional[str], Opt('--domain', '-d', help='Domain to login to.')] = None,
```

Comment:
I thought we decided to punt on configuring `domain` in this PR and address that later?